config/locales/en.yml

```diff
@@ -398,7 +398,8 @@ en:
         add_fixed_fee: 'Add another Fixed fee'
         add_misc_fee: 'Add another miscellaneous fee'
         expenses: 'Expenses'
-        expenses_help_text: 'Please enter details for each separate date'
+        expenses_help_text: Enter details and relevant dates for each separate expense
+        hours_help_text: Hours should be in decimal format (eg 1.25 hours for 1 hour and 25 mins) and will be rounded to the nearest quarter (eg 1.40 becomes 1.50)
```

Comment:
This is the copy she's seen and approved though.

On 15 December 2015 at 10:42, Edward Andress <
edward.andress@digital.justice.gov.uk> wrote:

> Actually that's a mistake (and I thought I had removed it but this ended
> up bouncing back and forth between Rachel and I so I lost track...); it
> should read "(eg 1.25 hours for 1 hour and 15 mins)"
>
> On 15 December 2015 at 10:36, Alistair Laing <notifications@github.com>
> wrote:
>
>> In config/locales/en.yml
>> <https://github.com/ministryofjustice/advocate-defence-payments/pull/842#discussion_r47620265>
>> :
>>
>> > @@ -398,7 +398,8 @@ en:
>> >          add_fixed_fee: 'Add another Fixed fee'
>> >          add_misc_fee: 'Add another miscellaneous fee'
>> >          expenses: 'Expenses'
>> > -        expenses_help_text: 'Please enter details for each separate date'
>> > +        expenses_help_text: Enter details and relevant dates for each separate expense
>> > +        hours_help_text: Hours should be in decimal format (eg 1.25 hours for 1 hour and 25 mins) and will be rounded to the nearest quarter (eg 1.40 becomes 1.50)
>>
>> I'm not sure about this copy sounds rather "techy". i.e what if someone
>> thinks 15mins is 0.25 (i.e a quarter)? the last eg is confused because
>> someone might read the 1.50 as being 1hr 30mins
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/ministryofjustice/advocate-defence-payments/pull/842/files#r47620265>
>> .
>>
>
>